docs/content/docs/connectors/table/filesystem.md

```diff
@@ -126,6 +126,21 @@ The following connector metadata can be accessed as metadata columns in a table
       <td><code>STRING NOT NULL</code></td>
       <td>Full path of the input file.</td>
     </tr>
+    <tr>
+      <td><code>filename</code></td>
+      <td><code>STRING NOT NULL</code></td>
+      <td>Name of the file, that is the farthest element from the root of the filepath.</td>
+    </tr>
+    <tr>
+      <td><code>size</code></td>
```

Comment:
`filename` vs `size` sounds inconsistent. Shall we call it `file.name`, `file.size`, `file.modification-time`? This would be more consistent with existing metadata and other option keys.

Reply:
or we keep it simply and do `name`, `size`, `modification-time`